packages/model-viewer/src/utilities.ts

```diff
@@ -198,6 +198,15 @@ export const isDebugMode = (() => {
        self.location.search.match(debugQueryParameter));
 })();
 
+/**
+ * @returns high-perofmance by default. However, developer has the option to set this.
+ * In that case we return developer's choice
+ */
+export const getPowerPreference = ():string => {
```

Comment:
I think you can remove this method since you're setting it to the default value as it's constructed. If it's really not defined when you need it, we should probably understand why.